src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;// радиус для круга
+    private static final int MAX_BOUND = 20;// максимальное число для фигуры
+    private static final int MIN_BOUND = 1;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle(Color.WHITE, DEFAULT_RADIUS);
+    }
+
+    public Figure getRandomFigure() {
+        FigureType figureType = FigureType.values()[random.nextInt(FigureType.values().length)];
+        switch (figureType) {
+            case CIRCLE:
+                return getRandomCircle();
+            case RECTANGLE:
+                return getRandomRectangle();
+            case SQUARE:
+                return getRandomSquare();
+            case TRAPEZOID:
+                return getRandomTrapezoid();
+            case TRIANGLE:
+                return getRandomTriangle();
+            default:
+                return null;
```

Comment:
```suggestion
                return getDefaultFigure();
```

[checklist:](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced) Don’t return null from a method.
Returning null from a method is a bad practice. If you use switch-case in your solution, you may just put the last possible option in the default case.